app/src/main/java/org/mozilla/fenix/historymetadata/HistoryMetadataService.kt

```diff
@@ -127,6 +136,18 @@ class DefaultHistoryMetadataService(
             storage.deleteHistoryMetadataOlderThan(olderThan)
         }
     }
+
+    override fun removeMetadata(groups: Set<HistoryMetadataGroup>) {
```

Comment:
Let's not add this to the service at all, we don't really need it here since with the new api this can just be a quick storage call up a few layers.